deployment/terraform/strings.go

```diff
@@ -136,6 +136,9 @@ net.ipv4.ip_local_port_range = 1024 65000
 net.ipv4.tcp_fin_timeout = 30
 `
 
+const agentCmdWithLog = `/bin/bash -c '/home/ubuntu/mattermost-load-test-ng/bin/ltagent server &>> /home/ubuntu/agent.log'`
+const agentCmdWithoutLog = `/home/ubuntu/mattermost-load-test-ng/bin/ltagent server`
```

Comment:
Can we have this as the base command and add the /bin/bash and &>> wrappers around it ? Less duplication that way.